.github/workflows/routine_l10n_obsolete.yml

```
@@ -1,7 +1,7 @@
 name: Bimonthly l10n cleanup reminder
 on:
   schedule:
-    - cron: 0 09 14 */2 *
+    - cron: 0 10 15 2-12/2 *
```

Hmh while it's pretty well explained in the workflow editing UI, just reading this doesn't make much sense. Would a comment help?

```suggestion
    - cron: 0 10 15 2-12/2 *  # runs every even month on the 15th day at 10 AM UTC
```